src/main/java/com/telefonica/iot/perseo/EventsServlet.java

```diff
@@ -76,25 +77,27 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         Utils.putCorrelatorAndTrans(request);
         logger.debug("events doPost");
         response.setCharacterEncoding("UTF-8");
-        response.setContentType("application/json;charset=UTF-8");
-        PrintWriter out = response.getWriter();
-        try {
-            StringBuilder sb = new StringBuilder();
+        response.setContentType("application/json;charset=UTF-8");        
+        try {        	
             String eventText = Utils.getBodyAsString(request);
-            logger.info("incoming event:" + eventText);
```

Comment:
I see a general change from string concatenation style (+) to format(). Why this change is neeed? Or, in other words, why usage of format() is prefereable over string concatenation?

Reply:
This is this is due to an issue of the maintainability axis that suggests using format() instead of string concatenation.

References:
https://wiki.sei.cmu.edu/confluence/display/c/FIO47-C.+Use+valid+format+strings

Reply:
Fine. NTC.